.github/security-insights.yml

```diff
@@ -0,0 +1,76 @@
+header:
+  schema-version: 2.0.0
+  last-updated: '2025-11-24'
+  last-reviewed: '2025-11-24'
+  url: https://github.com/ossf/scorecard
+  comment: This file contains the security information for the Scorecard project.
+
+project:
+  name: Scorecard
+  administrators:
+    - name: Stephen Augustus
+      affiliation: Bloomberg
+      primary: true
+    - name: Raghav Kaul
+      affiliation: Google
+    - name: Adam Korczynski
+      affiliation: ADA Logics
```

Comment:
the spec says these individuals have admin access to the project, which Adam currently doesn't

Reply:
Fixed in https://github.com/ossf/scorecard/pull/4863/commits/a6f2f21367fae315011c73909b375049c28a8033.